lib/Polaris.Class.ps1

```diff
@@ -338,32 +338,34 @@ class Polaris {
     }
 
     static [void] Send (
-        [System.Net.HttpListenerResponse]$RawResponse, 
-        [byte[]]$ByteResponse, 
-        [int]$StatusCode, 
-        [string]$ContentType, 
+        [System.Net.HttpListenerResponse]$RawResponse,
+        [byte[]]$ByteResponse,
+        [int]$StatusCode,
+        [string]$ContentType,
         [System.Net.WebHeaderCollection]$Headers
     ) {
-        $RawResponse.StatusCode = $StatusCode;
-        $RawResponse.Headers = $Headers;
-        $RawResponse.ContentType = $ContentType;
-        $RawResponse.ContentLength64 = $ByteResponse.Length;
-        $RawResponse.OutputStream.Write($ByteResponse, 0, $ByteResponse.Length);
-        $RawResponse.OutputStream.Close();
+        $RawResponse.StatusCode = $StatusCode
+        $RawResponse.Headers = $Headers
+        if ($ByteResponse.Length -gt 0) {
+            $RawResponse.ContentType = $ContentType
```

Comment:
Is there a reason you wrapped the ContentType specifically with an if check?

Reply:
If the response is empty there should be no Content-Type header. 

https://stackoverflow.com/questions/15032932/what-should-the-content-type-be-for-a-4xx-error-without-a-body

I thought the if at that layer would be a nice way to prevent that from getting set.